plugins/tiddlywiki/menubar/config.tid

```diff
@@ -1,6 +1,8 @@
-title: $:/plugins/tiddlywiki/menubar/config
-tags: $:/tags/ControlPanel/Toolbars
 caption: Menu Bar
+created: 20230519012218346
```

Comment:
We avoid created/modified fields on shadow tiddlers.

Reply:
Done in my next commit